sycl/ReleaseNotes.md

```diff
@@ -1,3 +1,227 @@
+   iterations for FPGA [f74b4ef]
+ - Impelemented generation of compiler report for kernel arguments [201f902]
+ - Implemented SYCL 2020 [sub-group](doc/extensions/SubGroup/SYCL_INTEL_sub_group.asciidoc#attributes)
+   size functionality [347e41c]
+ - Implemented SYCL 2020 interoperability API [e6733e4]
+ - Added [group sorting algorithm](doc/extensions/GroupAlgorithms/SYCL_INTEL_group_sort.asciidoc)
+   extension specification [edaee9b]
+ - Added [initial draft](doc/extensions/LevelZeroBackend/LevelZeroBackend.md)
+   for querying of free device memory in LevelZero backend extension [fa428bf]
+ - Added [InvokeSIMD](doc/extensions/InvokeSIMD/InvokeSIMD.asciidoc) and
+   [Uniform](doc/extensions/Uniform/Uniform.asciidoc) extensions [72e1611]
+ - Added [Matrix Programming Extension for DPC++ document](doc/extensions/Matrix/dpcpp-joint-matrix.asciidoc) [ace4c733]
+ - Implemented SYCL2020 `sycl::span` [9356d53]
+ - Added [device-if](doc/extensions/DeviceIf/device_if.asciidoc) extension
+   [4fb95fc]
+ - Added a [programming guide](doc/MultiTileCardWithLevelZero.md) for
+   multi-tile and multi-card under Level Zero backend [d581178a]
+
+## Improvements
+### SYCL Compiler
+ - Use `opencl-aot` instead of `aoc` when AOT flow for FPGA Emulator is
+   triggered [3a99558]
+ - Allowed for using an external host compiler [6f0ad1a]
+ - Cleaned up preprocessing output when `-fsycl` option is passed [3a18db6]
+ - Allowed kernel names in anonymous namespace [e47dbad]
+ - Set default value of `-sycl-std` to `2020` when  `-fsycl-is-device` or
+   `-fsycl-is-host` specified [680adc0]
+ - Added implication of `-fPIC` compilation for wrapped object when using
+   `-shared` [1754934]
+ - Added a diagnostic for `-fsycl` and `-ffreestanding` as non-supported
+   combination [a36c6720]
+ - [ESIMD] Renamed `simd::format` to `simd::bit_cast_view` [653dede1]
+ - Allowed `[[sycl::work_group_size_hint]]` to accept constant expr args
+   [ef8e4019]
+ - Deprecated the old-style SYCL attributes according to the SYCL 2020 spec
+   [001bbd42]
+ - Deprecated `[[intel::reqd_work_group_size]]` attribute spelling, please use
+   `[[sycl::reqd_work_group_size]]` instead [8ef7eacc]
+ - Enabled native FP atomics by default [0bbb68ee]
+ - Ignore `-O0` option for device code when compiling for FPGA with hardware
+   [7d94edf4]
+ - Allowed for known aliases to be used for `-fsycl-targets` [9778952a]
+ - [ESIMD] Added support for half type in ESIMD intrinsics [d5958ebf]
+ - Implemented `sycl::kernel::get_kernel_bundle` method [69a68a6d]
+ - Added a diagnostic in case of timing issues for FPGA AOT [c69a3115]
+ - Added support for C `memcpy` usages in the device code [76051ccf]
+ - [ESIMD] Added support for vectorizing scalar function [3fc66cc]
+ - Disable vectorization and loop transformation passes because loop unrolling
+   in "SYCL optimization mode" uses default heuristic, which is tuned for CPU
+   and might not be profitable for other devices [ff6929e6]
```

Comment:
This is to align with other items grammatically:
```suggestion
 - Disabled vectorization and loop transformation passes because loop unrolling
   in "SYCL optimization mode" used default heuristic, which tuned the code for
   CPU and might not have been profitable for other devices [ff6929e6]
```